NEWS.md

```diff
@@ -1,5 +1,14 @@
 ## 12.5.0 - Unreleased
 
+## 12.4.4 - Released Morning Glory 2022 R2 Hot Fix 1
+The focus of this release was to add copyright notice for FolioVertxCompletableFuture
+
+[Full Changelog](https://github.com/folio-org/mod-invoice/compare/v12.4.3...v12.4.4)
+
+### Stories
+* [FOLIO-3604](https://issues.folio.org/browse/FOLIO-3604) - FolioVertxCompletableFuture copyright violation
+
+
```

Comment:
Java doc has been updated previously in long live branch:
https://github.com/folio-org/mod-orders/blob/b12.4/src/main/java/org/folio/completablefuture/FolioVertxCompletableFuture.java

Reply:
yes correct